data/karthiknadar1204.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name":"Karthik Nadar",
+  "displayStatsPublic":true,
```

Comment:
We no longer need the `displayStatsPublic` key. So please remove this from your json. 

```suggestion
```